applications/audio/broadcast_configuration_tool/CMakeLists.txt

```
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(BCT)
+
+# Include application events and configuration headers
+zephyr_library_include_directories(app PRIVATE
+    ${CMAKE_CURRENT_SOURCE_DIR}/../../nrf5340_audio/include
```

```suggestion
    ${ZEPHYR_NRF_MODULE_DIR}/applications/nrf5340_audio/include
```
This way it doesn't break if the application is moved